corehq/apps/export/views/list.py

```diff
@@ -144,7 +144,7 @@ def _can_view(e, user_id):
                 and ('owner_id' in export and export['owner_id'] == self.request.couch_user.user_id) == my_exports
             ]
         if self.is_deid:
-            brief_exports = [x for x in brief_exports if x['is_safe']]  # TODO: is_deidentified
```

Comment:
:heart:

Reply:
FWIW, comments imply that `is_safe` can maybe be killed altogether: https://github.com/dimagi/commcare-hq/blob/dc382515afc266efd82a0237d011849e45e3ff76/corehq/apps/export/models/new.py#L707

I looked briefly at removing it altogether but wasn't sure what to do with code like this - https://github.com/dimagi/commcare-hq/blob/dc382515afc266efd82a0237d011849e45e3ff76/corehq/apps/export/custom_export_helpers.py#L132 - which implies that some exports do have an `is_safe` property...but maybe those are only old exports?